docs/tutorials/migrate-to-chiselled-rock.rst

```diff
@@ -0,0 +1,268 @@
+Migrate a popular Docker image to a Chiselled ROCK
+**************************************************
+
+Prerequisites
+-------------
+- snap enabled system (https://snapcraft.io)
+- LXD installed (https://linuxcontainers.org/lxd/getting-started-cli/)
+- skopeo installed (https://github.com/containers/skopeo)
```

Comment:
doesn't the tutorial use `skopeo` from the rockcraft snap?

Reply:
y we can delete this